### PR TITLE
Update rubygem-smart_proxy_host_reports to 0.0.5

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly
* 3.1 (if this is merged late)

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -674,7 +674,7 @@ foreman_proxy_plugin_packages:
     rubygem-smart_proxy_dns_powerdns: {}
     rubygem-smart_proxy_dns_route53: {}
     rubygem-smart_proxy_dynflow: {}
-    rubygem-smart_proxy_host_reports: {}
+    rubygem-smart_proxy_reports: {}
```
[the remark this answers is not in the document]
Comment (lzap) [reply package_manifest.yaml]: Fixed.

**Comment by lzap** [comment]
All green.

File: comps/comps-foreman-plugins-rhel7.xml [hunk 2]
```diff
@@ -321,7 +321,7 @@
       <packagereq type="default">tfm-rubygem-smart_proxy_dns_route53-doc</packagereq>
       <packagereq type="default">tfm-rubygem-smart_proxy_dynflow_core-doc</packagereq>
       <packagereq type="default">tfm-rubygem-smart_proxy_dynflow-doc</packagereq>
-      <packagereq type="default">tfm-rubygem-smart_proxy_host_reports-doc</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_reports-doc</packagereq>
```
[the remark this answers is not in the document]
Comment (lzap) [reply comps/comps-foreman-plugins-rhel7.xml]: Fixed.

**Review by ekohl — CHANGES_REQUESTED** [review]
It still isn't correctly sorted. Please use [./comps_doc.sh](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/comps_doc.sh) to sort comps XML files correctly.

This is what we use to keep tito.props sorted:
https://github.com/theforeman/foreman-packaging/blob/7280b49b9a35569970716e7516c285f68db7f9c3/add_gem_package.sh#L70-L76

For future reference, you can also use [./add_host.py](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/add_host.py) to correctly sort the manifest.

**Comment by lzap** [comment]
The script made zero relevant changes:

```
[lzap@nuc foreman-packaging-rpm]$ ./comps_doc.sh
[lzap@nuc foreman-packaging-rpm]$ gd
diff --git a/comps/comps-foreman-plugins-el8.xml b/comps/comps-foreman-plugins-el8.xml
index f6b08b050..12a5fb827 100644
--- a/comps/comps-foreman-plugins-el8.xml
+++ b/comps/comps-foreman-plugins-el8.xml
@@ -184,6 +184,7 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
+      <packagereq type="default">rubygem-acts_as_list-doc</packagereq>
       <packagereq type="default">rubygem-angular-rails-templates-doc</packagereq>
       <packagereq type="default">rubygem-aws-eventstream-doc</packagereq>
       <packagereq type="default">rubygem-aws-sigv4-doc</packagereq>
```

I manually sorted `tito.props`. If I use `sort -u` in vim, it shuffles things a bit and I do not want to introduce irrelevant changes.

**Comment by ekohl** [comment]
> I manually sorted `tito.props`. If I use `sort -u` in vim, it shuffles things a bit and I do not want to introduce irrelevant changes.

Sometimes it works to use line mode in vim. shift+v, select the lines and type `:sort`.

In this case it's probably easier to use this for comps:
```sh
./add_to_comps.rb comps/comps-foreman-rhel7.xml tfm-rubygem-smart_proxy_reports
./add_to_comps.rb comps/comps-foreman-el8.xml rubygem-smart_proxy_reports
# Remove old entries
./comps_doc.sh
```

**Comment by lzap** [comment]
Is it allright now? Builds are green.

**Comment by lzap** [comment]
I did git reset and executed the suggested commands, except these were supposed to be "comps-foreman-plugin-*.xml" instead. I hope it is now okay.

**Review by ekohl — APPROVED** [review]
Looks good. I haven't merged yet since this needs a bit of a koji dance. This approval is for @lzap to know I consider it ready.

**Pull request merged**